bootmenu-aarch64-jeos-20190315.json

```
@@ -15,6 +15,7 @@
     "ENV-UEFI-1",
     "ENV-LEAP-1",
     "ENV-VERSION-15.1",
-    "bootloader-grub2"
+    "bootloader-grub2",
+	"grub2"

```

the indentation looks wrong

```suggestion
      "grub2"
```

---

Fixed!